src/ontology/mondo-edit.obo

```diff
@@ -464918,6 +464920,2857 @@ is_a: MONDO:0005550 {source="https://orcid.org/0000-0002-5002-8648"} ! infectiou
+xref: OMIA:000616 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000616"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0002561 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog lysosomal storage disease
+
+[Term]
+id: MONDO:1010172
+name: alpha-mannosidosis, non-human animal
+def: "Alpha-mannosidosis that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000625 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000625"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000625"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009561 {source="OMIA:000625"} ! cross-species analog alpha-mannosidosis
+
+[Term]
+id: MONDO:1010173
+name: mucopolysaccharidosis IIID, non-human animal
+def: "Mucopolysaccharidosis IIID that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000665 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000665"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000665"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009658 {source="OMIA:000665"} ! cross-species analog mucopolysaccharidosis type 3D
+
+[Term]
+id: MONDO:1010174
+name: mucopolysaccharidosis VI, non-human animal
+def: "Mucopolysaccharidosis VI that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000666 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000666"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000666"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009661 {source="OMIA:000666"} ! cross-species analog mucopolysaccharidosis type 6
+
+[Term]
+id: MONDO:1010175
+name: mucopolysaccharidosis VII, non-human animal
+def: "Mucopolysaccharidosis VII that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000667 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000667"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000667"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009662 {source="OMIA:000667"} ! cross-species analog mucopolysaccharidosis type 7
+
+[Term]
+id: MONDO:1010176
+name: Niemann-Pick disease, type C1, non-human animal
+def: "Niemann-Pick disease, type C1 that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000725 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000725"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000725"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009757 {source="OMIA:000725"} ! cross-species analog Niemann-Pick disease, type C1
+
+[Term]
+id: MONDO:1010177
+name: mucopolysaccharidosis IIIA, non-human animal
```

Comment:
In human, "disease ~ type x" is based on the gene affected. Should we have only the species-specific term(s) instead of this generic term based on "general species unspecific" gene? 
Note that these species-specific diseases (based on species-specific genes) could be analogous to the human type x disease, but it is difficult to create an axiom referring to the general "species-not- specific" gene.